pikaur/build.py

```diff
@@ -375,6 +377,7 @@ def get_latest_dev_sources(
                     "\n".join((
```

Comment:
(to make possible re-showing prompt, the easiest way is to extract this code starting from `prompt = "{} {}\n{}\n> ".format(` into `recover_prompt()` func, so it could call itself recursively to re-show without action